### PR TITLE
DM-15045: Activate convolved measurements on undeblended sources

### PR DESCRIPTION
**Pull request by PaulPrice**

### CODE REVIEW

File: python/lsst/meas/extensions/convolved/convolved.py [hunk 1]
```diff
@@ -75,8 +75,8 @@ def __init__(self, name, schema, seeing, config, metadata):
         kronKeys = Struct(
             result=lsst.meas.base.FluxResultKey.addFields(schema, name + "_kron",
                                                           doc="convolved Kron flux: seeing %f" % (seeing,)),
-            flag = schema.addField(name + "_kron_flag", type="Flag",
-                                   doc="convolved Kron flux failed: seeing %f" % (seeing,)),
+            flag=schema.addField(name + "_kron_flag", type="Flag",
+                                 doc="convolved Kron flux failed: seeing %f" % (seeing,)),
```
Comment (TallJimbo) [inline python/lsst/meas/extensions/convolved/convolved.py]: Perhaps for next time, `schema.join(name, 'kron', 'flag')` would be the preferred way to concatenate schema name components these days.  We haven't made a push to use it across the board, though, so I won't insist on doing it on a high-priority fix ticket like this one.

File: python/lsst/meas/extensions/convolved/convolved.py [hunk 2]
```diff
@@ -478,7 +486,7 @@ def measureAperture(self, measRecord, exposure, aperturePhot):
         """
         try:
             aperturePhot.measure(measRecord, exposure)
-        except:
+        except:  # noqa E722: want to catch any and all errors
```
Comment (TallJimbo) [inline python/lsst/meas/extensions/convolved/convolved.py]: Still better to do `except Exception`, so you don't block e.g. `KeyboardInterrupt`, I think.

File: python/lsst/meas/extensions/convolved/convolved.py [hunk 3]
```diff
@@ -93,6 +93,13 @@ class BaseConvolvedFluxConfig(Config):
     maxSincRadius = Field(dtype=float, default=10.0,
                           doc="Largest aperture for which to use the sinc aperture code for Kron (pixels)")
     kronRadiusForFlux = Field(dtype=float, default=2.5, doc="Number of Kron radii for Kron flux")
+    registerForApCorr = Field(dtype=bool, default=True,
+                              doc="Register measurements for aperture correction?\n"
+                                  "The aperture correction registration is done at instantiation time\n"
+                                  "because the column names are derived from the configuration rather than\n"
+                                  "being static. Sometimes you want to turn this off, e.g., when you will\n"
+                                  "use aperture corrections derived from somewhere else through the 'proxy'\n"
+                                  "mechanism.")
```
Comment (TallJimbo) [inline python/lsst/meas/extensions/convolved/convolved.py]: I didn't know what "instantiation time" meant, exactly, when first reading this.  Maybe better to say "when the Plugin is instantiated".

File: tests/test_convolved.py [hunk 4]
```diff
@@ -203,28 +212,31 @@ def check(self, psfFwhm=0.5, flux=1000.0, forced=False):
...
+                # Kron succeeded and match expectation
+                if not forced:
+                    kronName = algConfig.getKronResultName(targetSeeing)
+                    kronApRadius = algConfig.kronRadiusForFlux*kronRadius
+                    self.assertFloatsAlmostEqual(source.get(prefix + kronName + "_flux"),
+                                                 expected(kronApRadius), rtol=1.0e-3)
+                    self.assertGreater(source.get(prefix + kronName + "_fluxSigma"), 0)
+                    self.assertFalse(source.get(prefix + kronName + "_flag"))
+
+                # Aperture measurements suceeded and match expectation
```
Comment (TallJimbo) [inline tests/test_convolved.py]: `suceeded` -> `succeeded`

File: setup.cfg [hunk 5]
```diff
@@ -0,0 +1,19 @@
+[flake8]
+max-line-length = 110
+ignore = E133, E226, E228, N802, N803, N806
+exclude =
+    __init__.py,
+    doc/conf.py,
+    python/lsst/afw/cameraGeom/cameraGeomLib.py,
```
Comment (timj) [inline setup.cfg]: You probably want to clean up these exclusions.

File: tests/test_convolved.py [hunk 6]
```diff
@@ -243,7 +255,7 @@ def setup_module(module, backend="virtualDevice"):
     lsst.utils.tests.init()
     try:
         afwDisplay.setDefaultBackend(backend)
-    except:
+    except:  # noqa E722: want to catch any and all errors
```
Comment (timj) [inline tests/test_convolved.py]: Please catch `BaseException`. It is no longer standard practice in python3 to have a bare except.

**Pull request merged**